### ISSUE TITLE
TypeError: Cannot read property 'generator' of undefined

### ISSUE THREAD

**Issue by micahstubbs** [nteract/semiotic#511]
version: `"semiotic": "2.0.0-beta.0",`

![Screen Shot 2019-10-31 at 4 28 01 PM](https://user-images.githubusercontent.com/2119400/67992835-7fbd8280-fbfb-11e9-8455-a73940919677.png)

```
×
TypeError: Cannot read property 'generator' of undefined
(anonymous function)
node_modules/semiotic/lib/VisualizationLayer.js:266
  263 | var pipe = renderPipeline[k];
  264 | 
  265 | if (pipe && (pipe.data && _typeof(pipe.data) === "object" && !Array.isArray(pipe.data) || pipe.data && pipe.data.length > 0)) {
> 266 |   console.log("sketchyRenderingEngine", sketchyRenderingEngine, sketchyRenderingEngine.generator);
      | ^  267 |   var additionalMarkProps = {
  268 |     sketchyGenerator: sketchyRenderingEngine && sketchyRenderingEngine.generator,
  269 |     "aria-label": pipe.ariaLabel && pipe.ariaLabel.items || "dataviz-element",
View compiled
VisualizationLayer.updateVisualizationLayer
node_modules/semiotic/lib/VisualizationLayer.js:262
  259 | var renderKeys = renderOrder.concat(renderVizKeys.filter(function (d) {
  260 |   return renderOrder.indexOf(d) === -1;
  261 | }));
> 262 | renderKeys.forEach(function (k) {
      | ^  263 |   var pipe = renderPipeline[k];
  264 | 
  265 |   if (pipe && (pipe.data && _typeof(pipe.data) === "object" && !Array.isArray(pipe.data) || pipe.data && pipe.data.length > 0)) {
View compiled
VisualizationLayer.componentWillMount
node_modules/semiotic/lib/VisualizationLayer.js:472
  469 | }, {
  470 |   key: "componentWillMount",
  471 |   value: function componentWillMount() {
> 472 |     this.updateVisualizationLayer(this.props);
      | ^  473 |   }
  474 | }, {
  475 |   key: "componentWillReceiveProps",
```

**Comment by micahstubbs** [comment]
I sadly do not have a minimal repro example yet.   will work on that.

**Comment by micahstubbs** [comment]
will try https://github.com/nteract/semiotic/releases/tag/2.0.0-beta.1 and see if I still experience this error 🤔

**Comment by micahstubbs** [comment]
beautiful, this bug is fixed in `semiotic@2.0.0-beta.1`

will close this issue and hope it helps future people who search for this.
